specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```
@@ -3003,9 +3003,14 @@
           "$ref": "#/definitions/AgentPoolMode"
         },
         "orchestratorVersion": {
+          "type": "string",
+          "title": "The version of Kubernetes specified by user.",
+          "description": "Both patch version <major.minor.patch> and alias minor version <major.minor> are supported. As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
+        },
+        "currentOrchestratorVersion": {
           "type": "string",
           "title": "The version of Kubernetes running on the Agent Pool.",
-          "description": "As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
+          "description": "If alias minor version is given (e.g. 1.22), the latest patch version supported of the input minor version will be applied (e.g. 1.22.7). Otherwise, it will be the same patch version as user specified."
```

Again IMO don't use `alias minor version` as it's an internal term and not one that users are necessarily going to understand. Maybe:
```suggestion
          "description": "If orchestratorVersion was a fully specified version <major.minor.patch>, this field will be exactly equal to it. If orchestratorVersion was <major.minor>, this field will contain the full <major.minor.patch> version being used."
```